.circleci/config.yml

```diff
@@ -42,6 +42,17 @@ jobs:
             else
               echo "build is skipped ..."
             fi
+      - run:
+          name: download all maven dependencies and groovy (second try)
+          command: |
+            RUN_JOB=`cat .ci-temp/run_job`
```

Comment:
`RUN_JOB` I do not like this name. It is more like a STATUS or EXIT_CODE to me.

Reply:
we share environment variables values through files not very intuitive approach, I agree, there might be better way ...
but it is just restoration of value from previous shell command.